traffic_monitor_golang/traffic_monitor/config/config.go

```diff
@@ -21,7 +21,6 @@ package config
 
```

Comment:
Should the TM changes go in a separate PR on which this one depends, or are you suggesting we branch the log package and keep a separate one for TM and TO?